src/content/self-hosted/administration/configuration.mdx

```diff
@@ -261,6 +265,8 @@ cloud:
 
 ##### IAM Role for Service Account (IRSA)
 
+Create a IAM role with access to the Okteto Registry bucket, bind the IAM role to the Okteto Registry Kubernetes service account and apply the changes below to your Helm configuration file:
```

Comment:
This reads like we are in a rush :P. We need to take our time and explain every step.  We don't have to write everything ourselves, we can send them to the corresponding docs on the cloud provider's doc. But we need to fully document the exact permissions needed for the role, people always get this wrong. 

The same goes for the GCP section.